src/bio/bio.pb-c.c

```diff
@@ -0,0 +1,144 @@
+       srv__bio_error_req__unpack
+                     (ProtobufCAllocator  *allocator,
+                      size_t               len,
+                      const uint8_t       *data)
+{
+  return (Srv__BioErrorReq *)
+     protobuf_c_message_unpack (&srv__bio_error_req__descriptor,
+                                allocator, len, data);
+}
+void   srv__bio_error_req__free_unpacked
+                     (Srv__BioErrorReq *message,
+                      ProtobufCAllocator *allocator)
+{
+  if(!message)
+    return;
+  assert(message->base.descriptor == &srv__bio_error_req__descriptor);
+  protobuf_c_message_free_unpacked ((ProtobufCMessage*)message, allocator);
+}
+static const ProtobufCFieldDescriptor srv__bio_error_req__field_descriptors[5] =
+{
+  {
+    "unmap_err",
+    1,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_BOOL,
+    0,   /* quantifier_offset */
+    offsetof(Srv__BioErrorReq, unmap_err),
+    NULL,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+  {
+    "update_err",
+    2,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_BOOL,
+    0,   /* quantifier_offset */
+    offsetof(Srv__BioErrorReq, update_err),
+    NULL,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+  {
+    "tgt_id",
+    3,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_INT32,
+    0,   /* quantifier_offset */
+    offsetof(Srv__BioErrorReq, tgt_id),
+    NULL,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+  {
+    "drpcListenerSock",
+    4,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_STRING,
+    0,   /* quantifier_offset */
+    offsetof(Srv__BioErrorReq, drpclistenersock),
+    NULL,
+    &protobuf_c_empty_string,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+  {
+    "instanceIdx",
+    5,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_UINT32,
+    0,   /* quantifier_offset */
+    offsetof(Srv__BioErrorReq, instanceidx),
+    NULL,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+};
+static const unsigned srv__bio_error_req__field_indices_by_name[] = {
+  3,   /* field[3] = drpcListenerSock */
+  4,   /* field[4] = instanceIdx */
+  2,   /* field[2] = tgt_id */
+  0,   /* field[0] = unmap_err */
+  1,   /* field[1] = update_err */
+};
+static const ProtobufCIntRange srv__bio_error_req__number_ranges[1 + 1] =
+{
+  { 1, 0 },
+  { 0, 5 }
+};
+const ProtobufCMessageDescriptor srv__bio_error_req__descriptor =
+{
+  PROTOBUF_C__MESSAGE_DESCRIPTOR_MAGIC,
+  "srv.BioErrorReq",
+  "BioErrorReq",
+  "Srv__BioErrorReq",
+  "srv",
+  sizeof(Srv__BioErrorReq),
+  5,
+  srv__bio_error_req__field_descriptors,
+  srv__bio_error_req__field_indices_by_name,
+  1,  srv__bio_error_req__number_ranges,
+  (ProtobufCMessageInit) srv__bio_error_req__init,
+  NULL,NULL,NULL    /* reserved[123] */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)